### PR TITLE
Fix undefined properties deprecated in PHP 8.2

### PR DESCRIPTION
**Pull request by alex-vlasov**
This is for 2.* version

### CODE REVIEW

File: phpseclib/Net/SSH1.php [hunk 1]
```diff
@@ -1418,7 +1434,7 @@ function getLog()
 
         switch (NET_SSH1_LOGGING) {
             case self::LOG_SIMPLE:
-                return $this->message_number_log;
```
Comment (terrafrost) [inline phpseclib/Net/SSH1.php:1421]: Should be `$this->protocol_flags_log`.

**Review by terrafrost — APPROVED** [review]
Just found one issue with it. I'll cherry pick this PR to the 1.0 branch and resolve that issue myself.

Thanks!

**Comment by terrafrost** [comment]
See https://github.com/phpseclib/phpseclib/commit/797351cd635177af249c65fdee8d541371592dba

I tagged a new release for the 2.0 branch as well - 2.0.41.

Normally Travis CI would have caught this but Travis CI doesn't yet support PHP 8.2. GitHub Actions does but not Travis CI.

Thanks!

**Comment by alex-vlasov** [comment]
Thanks for quick turnaround!

**Pull request closed without merging**